docs/data-sources/equinix_fabric_connection.md

```diff
@@ -13,7 +13,13 @@ Fabric V4 API compatible data resource that allow user to fetch connection for a
 
 ~> **Note** Equinix Fabric v4 resources and datasources are currently in Beta. The interfaces related to `equinix_fabric_` resources and datasources may change ahead of general availability. Please, do not hesitate to report any problems that you experience by opening a new [issue](https://github.com/equinix/terraform-provider-equinix/issues/new?template=bug.md)
```

Comment:
Should this beta notice be removed from these resources now or are there additional things that need to happen before the beta notice can be removed?

Reply:
It's ok to remove it now. We're out of the beta cycle and they're actively maintained in GA. I've made the updates 👍 

This PR should be good to go now.